src/main/java/macrobase/runtime/standalone/MacroBaseContextualBatchCommand.java

```diff
@@ -0,0 +1,39 @@
+package macrobase.runtime.standalone;
+
+import io.dropwizard.cli.ConfiguredCommand;
+import io.dropwizard.setup.Bootstrap;
+import macrobase.MacroBase;
+import macrobase.analysis.pipeline.BasicBatchedPipeline;
+import macrobase.analysis.pipeline.BasicContextualBatchedPipeline;
+import macrobase.analysis.result.AnalysisResult;
+import macrobase.analysis.result.ContextualAnalysisResult;
+import macrobase.conf.MacroBaseConf;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MacroBaseContextualBatchCommand extends ConfiguredCommand<MacroBaseConf> {
```

Comment:
Not necessary // remove. See https://github.com/stanford-futuredata/macrobase/pull/116